index.html

```diff
@@ -1,16 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
+    <header>
+      <img src="./images/logo_cake.svg" alt="logo" />
+      <div>
+        <nav class="menu">
+          &#9776;
+        </nav>
+        <h4>
+          The best cakes in town delivered to your door
+        </h4>
+      </div>
+    </header>
+
+    <main>
+      <section id="welcome">
+        <div class="welcome-text">
+          <h2>Welcome</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </div>
+        <div class="welcome-image">
+          <img src="./images/hero_cake.jpg" alt="welcome-img" />
+        </div>
+      </section>
+      <section id="cakes">
+        <div class="cake-div">
+          <img src="./images/cake1.jpg" alt="cake-img" />
+        </div>
+        <div class="cake-div">
+          <img src="./images/cake1.jpg" alt="cake-img" />
+        </div>
+        <div class="cake-div">
+          <img src="./images/cake1.jpg" alt="cake-img" />
+        </div>
+        <div class="cake-div">
+          <img src="./images/cake1.jpg" alt="cake-img" />
+        </div>
+      </section>
+    </main>
 
-</html>
\ No newline at end of file
+    <footer>
+      <div class="footer-divs">
+        <div class="first">
+          First Div
+        </div>
+        <div class="second">
+          Second Div
+        </div>
+        <div class="third">
+          Third Div
+        </div>
+      </div>
```

Comment:
Could you use a list here?

Reply:
I thought I can create container and List can be added inside div container when needs